### DISCUSSION TITLE
CssClass is not added when set the CssClass of AutoGenerateColumn in the model

### DISCUSSION THREAD

**Discussion by sunshuai3748** [dotnetcore/BootstrapBlazor#4473] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

CssClass is not added when set the CssClass of AutoGenerateColumn in the model

### Describe the solution you'd like

CssClass should be added to somewhere of the related controls

**Comment by bb-auto** [comment]
@sunshuai3748 Thank you for reporting. We will give triage later.

**Comment by ArgoZhang** [comment, marked as answer]
@sunshuai3748 `AutoGenerateColumn` 属性 `CssClass` 是给表格单元格内设置样式使用的

https://github.com/dotnetcore/BootstrapBlazor/blob/be300a878d63ead48a8775dc81d33f8fda8b5827/src/BootstrapBlazor/Components/Table/Table.razor.Sort.cs#L425


编辑组件如需要自定义样式可以如下设置
``` razor
<TableColumn @bind-Field="@context.Name" Width="100" ComponentParameters="ComponentParameters" />
```

`ComponentParameters` 为参数字典
